elixir/apps/api/lib/api/schemas/account_schema.ex

```diff
@@ -0,0 +1,72 @@
+defmodule API.Schemas.Account do
+  alias OpenApiSpex.Schema
+
+  defmodule LimitSchema do
+    require OpenApiSpex
+    alias OpenApiSpex.Schema
+
+    OpenApiSpex.schema(%{
+      title: "AccountLimit",
+      description: "Account limit with usage information",
+      type: :object,
+      properties: %{
+        used: %Schema{type: :integer, description: "Current usage count"},
+        available: %Schema{type: :integer, description: "Remaining available count"},
+        total: %Schema{type: :integer, description: "Total allowed count"}
+      },
+      required: [:used, :available, :total]
+    })
+  end
+
+  defmodule LimitsSchema do
+    require OpenApiSpex
+    alias OpenApiSpex.Schema
+
+    OpenApiSpex.schema(%{
+      title: "AccountLimits",
+      description: "Account limits and usage information",
+      type: :object,
+      properties: %{
+        users: API.Schemas.Account.LimitSchema,
+        monthly_active_users: API.Schemas.Account.LimitSchema,
+        service_accounts: API.Schemas.Account.LimitSchema,
+        account_admin_users: API.Schemas.Account.LimitSchema,
+        gateway_groups: API.Schemas.Account.LimitSchema
+      }
+    })
+  end
+
+  defmodule GetSchema do
```

Comment:
Was just wondering about this module name.  Was there a reason for going with `GetSchema` instead of `Schema` like in other entities?

Reply:
Yeah I noticed that too - I actually think this style makes slightly more sense because we often use different schemas for `GET` endpoints vs `PUT / POST / PATCH` endpoints. I can rename it though - wdyt?

Reply:
Let's have a quick chat. I think we're saying the same things in different accents